README.md

```diff
@@ -5,7 +5,7 @@ SPDX-License-Identifier: GPL-2.0-or-later
 
 # userstorage
 
-[![](https://img.shields.io/badge/pypi-v0.5.1-orange)](https://pypi.org/project/userstorage/)
+[![](https://img.shields.io/badge/pypi-v0.5.1-orange)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/badge/python-2.7%20%7C%203.6%20%7C%203.7%20%7C%203.8%20%7C%203.9%20%7C%203.10-blue)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/badge/CI-passing-brightgreen)](https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml)
```

Comment:
Like previous commit, this is partial work in progress and should not be part of the pull request.